node-fs-readfile/cat.js

```diff
@@ -0,0 +1,15 @@
+import { readFile } from 'node:fs/promises';
+
+const files = process.argv.splice('2');
```

Comment:
This works? I'm actually surprised (JavaScript magic, I guess -- something I really dislike about JavaScript!). You are passing the number 2 as a string here. It should be passed as a number.